frontend/src/app/core/global_search/input/global-search-input.component.ts

```diff
@@ -57,6 +57,11 @@ import { ApiV3Service } from '../../apiv3/api-v3.service';
 
 export const globalSearchSelector = 'global-search-input';
 
+interface SearchResultItems {
+  items:SearchResultItem|SearchOptionItem[];
```

Comment:
Should both options be an array? So like this `SearchResultItem[]|SearchOptionItem[]`? Or am I missing something here?

Reply:
Hi @HDinger yes both should be array. I had written it wrongly.